.gitignore

```diff
@@ -1,3 +1,5 @@
 /node_modules
 /.vuepress/dist
 /yarn-error.log
+/.idea
```

Comment:
Please remove the additions here in the `.gitignore`. These should be in your global `.gitignore` file, not in the project's `.gitignore`.

Reply:
Thank you for the recommendation. 
But do You think it's good to define them in the ignore, because a lot of people (in this case, me too) are used to it being in the project.
I may have inherited it from laravel and other projects that have it included.

[Ex.: Laravel Repository](https://github.com/laravel/laravel/blob/586b9e7bf5efef4d205552cc285a3f8498767578/.gitignore)

Reply:
@lindyhopchris  Changes are already pushed.